static/js/scroll_bar.js

```diff
@@ -46,7 +46,6 @@ export function initialize() {
         $(".fixed-app").css("left", "-" + sbWidth + "px");
         $(".fixed-app .column-middle").css("margin-left", 7 + sbWidth + "px");
 
-        $(".column-right").css("right", sbWidth + "px");
```

Comment:
Is it intentional that this commit is in this PR?

Reply:
No @timabbott I will remove it